### PR TITLE
Refactor/remove asset transactions

### PR DESCRIPTION
**Pull request by zl730**
# PR Details

## Description

- remove transactions & related states & related apis:
   - issue
   - transfer
   - reissue
   - burn
   - exchange
- remove codes related to asset
- refactoring transaction parser
## Related Issue

None

## Motivation and Context

feature of asset may be duplicated by feature of token, which just released

## How Has This Been Tested

unit tests passed

## Types of changes

- [x] refactoring

## Checklist

- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ncying** [comment]
btw, since several transactions are removed, integral test will cause some error when compiling

File: src/main/scala/vsys/blockchain/transaction/Transaction.scala [hunk 1]
```diff
@@ -43,3 +43,13 @@ object Signed {
       validateSignatures(descendant).isLeft
     }.fold[E[S]](Right(s))(sd => Left(InvalidSignature(s, Some(validateSignatures(sd).left.get))))
 }
+
+trait NonFeeTransaction extends Transaction {
+  val transactionFee: Long = 0
+  val feeScale: Short = 100
+}
+
+trait AmountInvoved {
```
[the remark this answers is not in the document]
Comment (zl730) [reply src/main/scala/vsys/blockchain/transaction/Transaction.scala]: this trait is for decreasing duplicates between payment and lease
for example:
```scala
val amount =  tx match {
   case ptx: PaymentTransaction => ptx.amount
   case ltx: LeaseTransaction => ltx.amount
   case _ => 0
}
```
will be shorten to 
```scala
val amount =  tx match {
   case atx: AmountInvoved => atx.amount
   case _ => 0
}
```

[the remark this answers is not in the document]
Comment (ncying) [reply src/main/scala/vsys/blockchain/transaction/Transaction.scala]: ic

**Comment by zl730** [comment]
> btw, since several transactions are removed, integral test will cause some error when compiling

I think the integration test should be rewrite completely

**Comment by ncying** [comment]
sure

**Pull request merged**